### PR TITLE
Ratom 50 build continuous deploy

### PR DESCRIPTION
**Pull request by JeremyGibson**

### CODE REVIEW

File: input.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+! TRAVIS input file
+! Created with TRAVIS version compiled at Jan 14 2018 01:39:14
+! Input file written at Mon Jan 13 08:30:56 2020.
```
Comment (copelco) [inline input.txt:3]: What is this?

File: ratom_api/settings/local.example.py [hunk 2]
```diff
@@ -1,3 +0,0 @@
-from ratom_api.settings.dev import *   # noqa
```
Comment (copelco) [inline ratom_api/settings/local.example.py:1]: Why was this file deleted?

File: ratom_api/settings/base.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
 
 BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
 PROJECT_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.pardir))
+SECRET_KEY = os.getenv('SECRET_KEY')
```
Comment (copelco) [inline ratom_api/settings/base.py]: This differs from ``DJANGO_SECRET_KEY`` in the deploy settings. Is that intentional? Or does this need to be in base?

**Review by copelco — CHANGES_REQUESTED** [review]

File: input.txt (hunk 1 above)
Comment (JeremyGibson) [reply input.txt:3]: Something i added just to make a change.  will delete

File: ratom_api/settings/base.py (hunk 3 above)
Comment (JeremyGibson) [reply ratom_api/settings/base.py]: This can come out too. Part of me trying to figure out how to get passed the testing of the image.

File: ratom_api/settings/local.example.py (hunk 2 above)
Comment (JeremyGibson) [reply ratom_api/settings/local.example.py:1]: Not sure. I might have been trying things out. I'll put it back.

**Review by copelco — APPROVED** [review]
